ocrd_network/ocrd_network/database.py

```diff
@@ -35,6 +37,25 @@ async def sync_initiate_database(db_url: str):
     await initiate_database(db_url)
 
 
+async def db_create_workspace(mets_path: str) -> DBWorkspace:
+    """ Create a workspace-database entry only from a mets-path
+    """
+    if not Path(mets_path).exists():
+        raise ValueError(f'Cannot create DB workspace entry, `{mets_path}` does not exist!')
+    try:
+        return await db_get_workspace(workspace_mets_path=mets_path)
+    except ValueError:
```

Comment:
Why does this (potentially) raise a ValueError?

Reply:
When neither `workspace_mets_path` nor `workspace_id` matches some database document representing the Workspace.